app/concerns/callable.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# http://weblog.jamisbuck.org/2007/1/17/concerns-in-activerecord
+module Callable
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def call(*args)
+      new(*args).call
```

Comment:
Out of curiosity, what is which class would the `new` be invoking?

Reply:
Hm, not sure I fully understand the question, but essentially this `Callable` concern is being mixed in/included into every Service Object so that they can be invoked like: `CreateQuestion.call` rather than needing `CreateQuestion.new.call`. Let me know if that makes sense!

Reply:
Ah! I see! Thanks for understanding my question! You nailed it I realized my typo!